JeMPI_Apps/JeMPI_UI/src/utils/constants.ts

```diff
@@ -92,5 +93,24 @@ export const AUDIT_TRAIL_COLUMNS: GridColDef[] = [
     disableColumnMenu: true,
     headerClassName: 'super-app-theme--header',
     flex: 1
+  },
+  {
+    field: 'score',
+    headerName: 'Matching Type',
+    valueFormatter: ({ value }) => extractMatchingRule(value),
+    sortable: false,
+    disableColumnMenu: true,
+    headerClassName: 'super-app-theme--header',
+    flex: 1
   }
 ]
+
+const extractMatchingRule = (score: number) => {
```

Comment:
Hi @NyashaMuusha ,

As mention in the previous PR, score does not determine if something is matched Deterministically or Probablistically. It just happens that when we link Deterministically the score it 1.0 (i.e although quite rare, you could have a scenario where you have a score of 1.0 but linked Probablistically). 

What you want to try do is something like in the `calcNormalizedScore` method in `JeMPI_Apps/JeMPI_Linker/src/main/java/org/jembi/jempi/linker/backend/LinkerUtils.java`

Once you you obtain if it is linking Deterministically or Probablistically, update the `WorkCandidate` in `JeMPI_Apps/JeMPI_Linker/src/main/java/org/jembi/jempi/linker/backend/LinkerDWH.java` with these detail (in addition to the score)

When it calls `createInteractionAndLinkToExistingGoldenRecord` you will need to pass through these details